scripts/make-env

```diff
@@ -31,10 +33,16 @@ randstr() {
 }
 
 b64encode() {
-  cat "$@" | base64 --wrap=0 2>/dev/null || cat "$@" | base64 --break=0
+    echo "$@" | base64 --wrap=0 2>/dev/null || echo "$@" | base64 --break=0 2>/dev/null
 }
 
-VPN_CONFIG=$(b64encode <<STR
+b64file() {
+  # local FILE=$(awk 'FNR==1{print ""}1' $*)
+  # b64encode "$FILE"
+  b64encode "$(cat "$@")"
+}
+
+VPN_CONFIG=$(cat <<_EOF_
```

Comment:
Why change to `_EOF_` here? `STR` is used as the here string below, why not leave them the same?